kie-drools-wb/kie-drools-wb-webapp/pom.xml

```diff
@@ -523,6 +523,13 @@
       <artifactId>kie-wb-common-workbench-backend</artifactId>
     </dependency>
 
+    <!-- Drools Workbench Services -->
+    <dependency>
+      <groupId>org.drools</groupId>
+      <artifactId>drools-wb-verifier-api</artifactId>
+      <scope>provided</scope>
```

Comment:
Sure, no worries, just wanted to double check it is intended (if it would be to stay, it could use a comment explaining the situation...)